packages/react/src/primitives/Loader/Loader.tsx

```diff
@@ -34,6 +35,13 @@ const LoaderPrimitive: Primitive<LoaderProps, 'svg'> = (
   percentage = Math.max(percentage, 0);
 
   const percent = `${percentage}%`;
+  const componentClasses = classNames(
+    ComponentClassNames.Loader,
+    classNameModifier(ComponentClassNames.Loader, size),
+    classNameModifier(ComponentClassNames.Loader, variation),
+    isDeterminate ? ComponentClassNames.LoaderDeterminate : null,
```

Comment:
Use `classNameModifierByFlag`?

Reply:
The classNameModifier function is used to add modifiers onto the classnames, here we don't want to modify the classname instead just add the classname in if it `isDeterminate`